flytepropeller/pkg/controller/handler.go

```diff
@@ -89,6 +91,11 @@ func SetDefinitionVersionIfEmpty(wf *v1alpha1.FlyteWorkflow, version v1alpha1.Wo
 	}
 }
 
+func IsAborted(w *v1alpha1.FlyteWorkflow) bool {
```

Comment:
can this be moved to the same place IsDeleted is defined?

Reply:
Sure I will do it, tomorrow

Reply:
done!